src/main/java/com/example/backendspringboottechiteasycontroller/models/Television.java

```diff
@@ -0,0 +1,202 @@
+package com.example.backendspringboottechiteasycontroller.models;
+
+import jakarta.persistence.*;
+
+import java.util.ArrayList;
+import java.util.List;
+@Entity
+@Table(name="cars")
+public class Television {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer ID;
+    @Column(name="type")
```

Comment:
This name is the same as the variable name. If the DB column name can be equal to the variable name, than you don't need to alter it.